main.go

```diff
@@ -1,21 +1,18 @@
-//go:generate go-bindata -mode 444 -modtime 1 cert/cert.pem cert/key.pem openapi/openapi/fixtures3.json openapi/openapi/spec3.json
+//go:generate go-bindata -mode 0444 -modtime 1 cert/cert.pem cert/key.pem
+//go:generate go-bindata -o server/bindata.go -pkg server -mode 0444 -modtime 1 openapi/openapi/fixtures3.json openapi/openapi/spec3.json
```

Comment:
does this mean we should delete `bindata.go` at the root? `server/bindata.go` is the new path?

Reply:
Saving the comment for history here... The reason there are 2 bindata files is one of them is for fixtures/spec the other one is for the certs:
- server dir - includes fixtures3.json and spec3.json
- main dir - includes cert.pem and key.pem